docs/source/app_developers_guide/creating_sawtooth_network.rst

```diff
@@ -237,7 +237,98 @@ Step 4: Confirm Network Functionality
         MyKey: 999
 
 
-Step 5: Stop the Sawtooth Network (Optional)
+.. _configure-txn-procs-docker-label:
+
+Step 5. Configure the Allowed Transaction Processors (Optional)
+---------------------------------------------------------------
+
+By default, a validator accepts transactions from any transaction processor.
+However, Sawtooth allows you to limit which transaction processors can submit
```

Comment:
The transaction processors do not do the submitting. It would be more accurate to say "Sawtooth allows you to limit the types of transactions that can be submitted."